recipes/dealii/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dealii" %}
+{% set version = "9.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://dealii.kyomu.43-1.org/downloads/dealii-{{ version }}.tar.gz
+  sha256: df2f0d666f2224be07e3741c0e8e02132fd67ea4579cd16a2429f7416146ee64
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+  #run:
+  #  - python
+
+test:
+  requires:
+  - cmake
+  source_files:
+  - build
+  commands:
+  - 'cd build && make test'
+
+about:
+  home: https://dealii.org/
+  license: LGPL-2.1
+  license_file: LICENSE.md
+  summary: 'The deal.ii finite element package'
+
+  description: |
+    deal.II is a C++ program library targeted at the computational solution of
+    partial differential equations using adaptive finite elements. It uses
+    state-of-the-art programming techniques to offer you a modern interface to
+    the complex data structures and algorithms required.
+  doc_url: https://dealii.org/developer/doxygen/deal.II/Tutorial.html
+  dev_url: https://github.com/dealii/dealii
+
+extra:
+  recipe-maintainers:
+    - inducer
+    - xywei
```

Comment:
@xywei Are you happy to be listed as a maintainer of this recipe?

Reply:
Yes I am.